example-document/03-troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+# Troubleshooting {#troubleshooting}
+
+Many pipeline failures are caused by justa few issues. This section describes the debugging process of a failed pipeline.
+
+First of all, to see the pipeline statu:
+1. Go to your repository
+2. Open Actions tab
+3. Open detail of your pipeline run
+4. Find out what job is failing
+5. See detailed logs of failed job
+6. Seach for commonly known error messages using native GitHub log search on top right corner
+
+## Validate YAML documents
+
+YAML formatting is strict, and all errors in this job are related to it. Search for `yaml.scanner.ScannerError` to see what went wrong. Example of the log:
+
+```
+yaml.scanner.ScannerError: mapping values are not allowed in this context
+  in "/__w/istqb-ctal-ta/syllabus/metadata.yml", line 3, column 7
+```
+To resolve these issues:
+1. Navigate to affected file (`syllabus/metadata.yml` line 3, position 7 in this case)
+2. Resolve the formatting issues which could be cause by several things. The most common are:
+  * wrong indents - fix by adding 2 spaces per indent level
+  * using colon `:` or hashtag `#` in the text - fix by adding the text in quotes `"` or `'`
+3. Use online YAML linters like [yamllint.com](https://www.yamllint.com/) to test your YAML file
+
+## Produce PDF documents
+
+### Undefined references and citations
+
+The most common failure is one related to references. Search for `Undefined refs and citations` to see what went wrong. Example of the log:
+
+```
+	Latexmk: ====Undefined refs and citations with line #s in .tex file:
+	  Reference `section:analyze-sut-to-determine-the-appropriate-tas'
+	    on page 21 undefined on input line 37
+	  Reference `section:apply-layering-of-taf'
+	    on page 22 undefined on input line 38
+	  Reference `section:apply-design-principles-and-design-patterns-in-ta'
+	    on page 24 undefined on input line 38
+	  Reference `section:apply-different-approaches-for-automating-tcs'
+	    on page 47 undefined on input line 43
+	  Reference `section:apply-layering-of-taf'
+	    on page 48 undefined on input line 43
+	  Reference `section:explain-which-factors-support-and-affect-tas-maintainability'
+	    on page 50 undefined on input line 43
+	```
```

Comment:
This does not properly close the code block:

![image](https://github.com/user-attachments/assets/1a80bbef-28c1-442a-932b-1cd886705322)

To fix this, you should remove the indent to align the closing ticks with the opening ticks.